tests/system_command/system_command.slt

```diff
@@ -15,3 +15,32 @@ query T
 select read("${__TEST_DIR__}/test.txt")
 ----
 1919810
+
+# Note: it ends with 2 newlines
+system ok
+echo "114514"
+----
+114514
+
+
+# Note: when substitution is on, we need to escape the backslash
+# Note: 1 blank line in the middle is ok, but not 2
```

Comment:
What if the system command really contain 2 consecutive blank lines? 😕

Reply:
ntfs